components/ILIAS/Notifications/classes/Provider/NotificationsPushProvider.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\Notifications\Provider;
+
+use ILIAS\Notifications\ilNotificationPushHandler;
+use ILIAS\Notifications\Interfaces\PushProviderInterface;
+use ILIAS\Notifications\Model\ilNotificationConfig;
+use ILIAS\Notifications\Model\ilNotificationLink;
+use ILIAS\Notifications\Model\ilNotificationObject;
+use ILIAS\Notifications\Model\Push\PushQueueResult;
+use ilObjUser;
+
+abstract class NotificationsPushProvider implements PushProviderInterface
+{
+    protected const int TTL = 60;
+    protected ilNotificationPushHandler $handler;
```

Comment:
```suggestion

    protected ilNotificationPushHandler $handler;
```

Reply:
@mjansenDatabay Is this a psr convention? I am confused since this is by far not the only occurrence of this case in this PR and in ILIAS in general.

Personally, I prefer the compact patter of properties, but if obliged to change I would do this for all occurrences.

Greetings,
@iszmais